certbot/tests/display/completer_test.py

```diff
@@ -47,6 +51,8 @@ def test_complete(self):
         completion = my_completer.complete(self.tempdir, num_paths)
         self.assertEqual(completion, None)
 
+    @pytest.mark.skipif('readline' not in sys.modules,
+                        reason='Not relevant if readline is not available.')
```

Comment:
Unlike in [acme](https://github.com/certbot/certbot/blob/v0.27.1/acme/setup.py#L83), we're actually not currently requiring `pytest` to be installed to run tests for Certbot. We can change that if we need to, but instead, can we just use `unittest.skipIf` here?

Reply:
Done.